jetbrains-core/src/software/aws/toolkits/jetbrains/ui/feedback/FeedbackDialog.kt

```diff
@@ -90,8 +90,9 @@ class FeedbackDialog(val project: Project, initialSentiment: Sentiment = Sentime
             }.bind({ sentiment }, { sentiment = it })
 
             row(message("feedback.comment.textbox.title")) {}
+            row { comment(message("feedback.customer.alert.info")) }
             row {
-                comment = textArea().rows(6).columns(50).bindText(::commentText).applyToComponent {
+                comment = textArea().rows(6).columns(52).bindText(::commentText).applyToComponent {
```

Comment:
instead of hardcoding the length of this string, just add a `<br/> to the localized string

Reply:
Even though broke the string, ```textArea``` doesn't aligning with a string correctly. So, hardcoded to make align.